### DISCUSSION TITLE
get commad 503

### DISCUSSION THREAD

**Discussion by wangqingpen** [edgexfoundry/community#242] [Q&A]
I can use the following link to get the service  curl http://localhost:59882/api/v2/device/all 
![image](https://github.com/edgexfoundry/community/assets/72652818/73243a2f-745d-4fa9-8564-34a0dcba355f)
but i use   curl http://localhost:59882/api/v2/device/name/dlt698/cuurent
![image](https://github.com/edgexfoundry/community/assets/72652818/da421a91-cbb2-473f-be2c-34bfdede0795)
i look cuurent in all

**Comment by cloudxxx8** [comment]
please use the file or plan text instead of the command line screen shot.  It's hard to read.
for the error 503, please make sure your device service is reachable
